src/Formatter/PolicyFormatter.php

```diff
@@ -29,6 +29,12 @@ public function __toString(): string
 
                 return "{$directive->name()}=({$formattedRules})";
             })
+            ->when(
+                config('feature-policy.reporting.enabled'),
+                function (Collection $collection) {
+                    $collection->add('report-to=violation-reports');
```

Comment:
if reporting is enabled, then the Permission-Policy needs an additional directive to instruct the browser to report the prior directives.